_zigbee/Hama_00176592.md

```diff
@@ -13,5 +13,5 @@ link2: https://www.idealo.de/preisvergleich/OffersOfProduct/201561703_-starter-s
 compatible: [z2m, iob, zha, deconz, z4d]
 deconz: 6859
 z2m: SEA801-Zigbee/SEA802-Zigbee
-pairing: Rotate wheel button until display shows "OF". Press and hold button for 5 seconds. Display will show "--" when in pairing mode.
```

Comment:
while this might look like a typo, the display has only two eight-segment digits. it will display "OF".